sdk/tables/azure-data-tables/azure/data/tables/aio/_table_client_async.py

```diff
@@ -186,12 +186,12 @@ async def get_table_access_policy(self, **kwargs) -> Mapping[str, Optional[Table
 
     @distributed_trace_async
     async def set_table_access_policy(
-        self, signed_identifiers: Mapping[str, Optional[TableAccessPolicy]], **kwargs
+        self, signed_identifiers: Dict[str, Optional[TableAccessPolicy]], **kwargs
```

Comment:
Why did we need this change? `Mapping` should be supported here?

Reply:
I changed here bc I found it's `dict` in docstring.

Reply:
mypy doesn't complaing with `Mapping`. I think the reason I'm changing here is because it's `Dict` in the sync one's signature. I'll update both sync and async to type with `Mapping`.

Reply:
Updated.